### ISSUE TITLE
Zombie Pigman Spawner

### ISSUE THREAD

**Issue by Nights-Path** [Shadows-of-Fire/Apotheosis#132]
Set up a spawner killed my share of pigman, used an egg on the spawner........ Aaaaaaaand I get chickens.
I kill more pigman, get another egg, use it on the spawner.................... Aaaaaaaaaaaand more chickens.
I didn't get what was going on, then looked in my chest at the drops, then played with sound (Yes, I play muted) I heard the pigman, and one of the chickens BIT ME!
Long story short, the pigman spawner is trying to falsely spawn chicken jockies and if the spawn fails, it only spawns the chicken. Even if the spawn somehow succeeds, the pigman are invisible, jocky or not.

TLDR;
Pigman spawners are broken

**Comment by Shadows-of-Fire** [comment]
determined to be a bug in VTweaks
